sql/core/src/test/resources/test-data/arrow/boolData.json

```diff
@@ -0,0 +1,32 @@
+{
+    "schema": {
+        "fields": [
+            {
+                "name": "a_bool",
+                "type": {"name": "bool"},
+                "nullable": false,
+                "children": [],
+                "typeLayout": {
+                    "vectors": [
+                        {"type": "VALIDITY", "typeBitWidth": 1},
+                        {"type": "DATA", "typeBitWidth": 8}
+                    ]
+                }
+            }
+        ]
+    },
+
+    "batches": [
+        {
+            "count": 4,
+            "columns": [
+                {
+                    "name": "a_bool",
+                    "count": 4,
+                    "VALIDITY": [1, 1, 1, 1],
+                    "DATA": [true, true, false, true]
+                }
+            ]
+        }
+    ]
+}
```

Comment:
In the future, you may wish to generate these files rather than storing them in source control. The particulars of the Arrow JSON format for integration testing could change